platform/linux-generic/include/odp/api/plat/packet_types.h

```diff
@@ -156,7 +157,12 @@ typedef union {
 	uint32_t all_flags;
 
 	struct {
-		uint32_t reserved1:     12;
+		uint32_t reserved1:     11;
+
+	/*
+	 * Init flags
+	 */
+		uint32_t user_ptr_set:   1; /* Ptr has been set vs. NULL */
```

Comment:
The "vs" comment is confusing. "User ptr has been set" seems sufficient here.

Reply:
OK.